client/src/selectors/authSelectors.ts

```diff
@@ -3,3 +3,11 @@ import { RootStateOrAny } from "react-redux";
 export const getIsAuthenticated = () => {
   return (state: RootStateOrAny) => state.auth.isAuthenticated;
 };
+
+export const getUserName = () => {
+  return (state: RootStateOrAny) => state.auth.user.username;
+};
+
+export const getUser = () => {
+  return (state: RootStateOrAny) => state.auth.user;
+};
```

Comment:
You can return a value, not a function and use it like:

`const name = useSelector(getUserName)`

Reply:
done